packages/xrpl/test/integration/index.ts

```diff
@@ -1,62 +1 @@
-/* eslint-disable import/export -- Tells webpack which files exist. */
-
-// These go first because they're affected by the `ledger_accept`s
-export * from './transactions/escrowFinish.test'
-export * from './transactions/escrowCancel.test'
-
-// Transactions
-export * from './transactions/accountSet.test'
-export * from './transactions/checkCancel.test'
-export * from './transactions/checkCash.test'
-export * from './transactions/checkCreate.test'
-export * from './transactions/depositPreauth.test'
-export * from './transactions/escrowCreate.test'
-export * from './transactions/offerCancel.test'
-export * from './transactions/offerCreate.test'
-export * from './transactions/payment.test'
-export * from './transactions/paymentChannelClaim.test'
-export * from './transactions/paymentChannelCreate.test'
-export * from './transactions/paymentChannelFund.test'
-export * from './transactions/signerListSet.test'
-export * from './transactions/trustSet.test'
-
-// Requests
-export * from './requests/accountChannels.test'
-export * from './requests/accountCurrencies.test'
-export * from './requests/accountInfo.test'
-export * from './requests/accountLines.test'
-export * from './requests/accountObjects.test'
-export * from './requests/accountOffers.test'
-export * from './requests/accountTx.test'
-export * from './requests/bookOffers.test'
-export * from './requests/channelVerify.test'
-export * from './requests/depositAuthorized.test'
-export * from './requests/fee.test'
-export * from './requests/gatewayBalances.test'
-export * from './requests/ledger.test'
-export * from './requests/ledgerClosed.test'
-export * from './requests/ledgerCurrent.test'
-export * from './requests/ledgerData.test'
-export * from './requests/ledgerEntry.test'
-export * from './requests/noRippleCheck.test'
-export * from './requests/pathFind.test'
-export * from './requests/ripplePathFind.test'
-export * from './requests/serverInfo.test'
-export * from './requests/serverState.test'
-export * from './requests/submit.test'
-export * from './requests/submitMultisigned.test'
-export * from './requests/subscribe.test'
-export * from './requests/tx.test'
-export * from './requests/utility.test'
-
-export * from './fundWallet.test'
-export * from './integration.test'
-export * from './onConnect.test'
-export * from './regularKey.test'
-export * from './submitAndWait.test'
-
-// Because this does 256 ledger accepts, we do it last
-export * from './transactions/accountDelete.test'
-
-// Ensure you export all added tests above "export * from './finalTest'", otherwise they will not be run.
-export * from './finalTest.test'
+/* eslint-disable import/unambiguous -- silence is golden */
```

Comment:
If the file is empty, why does it need to exist?

Reply:
If it wasnt empty it errored and the webpack needed an entry point.  Might be worth investigating why that causes problems